normandy/conftest.py

```diff
@@ -0,0 +1,77 @@
+# Configuration file for running our tests
+import pytest
+
+from firefox_puppeteer import Puppeteer
+from marionette_driver.marionette import Marionette
+
+TIMEOUT = 20
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--admin-url",
```

Comment:
> 
> I don't think this option is used anywhere, though I can certainly image it being
> useful in the future.

That is a relic from when I was trying to get Marionette tests with the admin UI working.